kubectl-fdb/cmd/version.go

```diff
@@ -32,6 +32,7 @@ import (
 )
 
 var pluginVersion = "latest"
+var versionMessage = ""
```

Comment:
Can we remove this global variable and instead return an error in `usingLatestPluginVersion` with the according message?